src/js/common/schemaform/FormApp.jsx

```diff
@@ -136,6 +143,9 @@ class FormApp extends React.Component {
       const saveEnabled = !this.props.formConfig.disableSave;
       if (saveEnabled && (isSaved || isPrefill)) {
         props.fetchInProgressForm(currentForm, props.formConfig.migrations, isPrefill);
+        if (trimmedPathname.endsWith('resume')) {
+          props.router.replace(currentPagePath);
```

Comment:
This path is `undefined`, so it throws an error and doesn't seem to do anything. We do want to replace and not push to the history when we're on the resume page, so I think this logic needs to be somewhere in componentWillReceiveProps, where we make the router.push calls after successfully loading a form.